man/en/sos-upload.1

```diff
@@ -0,0 +1,111 @@
+.TH UPLOAD 1 "July 2024"
+
+.SH NAME
+sos_upload \- Upload files like previously generated sos reports or logs to a policy specific location
+.SH SYNOPSIS
+.B sos upload FILE [options]
+          [--case-id id]\fR
+          [--upload-url url]\fR
+          [--upload-user user]\fR
+          [--upload-pass pass]\fR
+          [--upload-directory dir]\fR
+          [--upload-method]\fR
+          [--upload-no-ssl-verify]\fR
+          [--upload-protocol protocol]\fR
+
+.PP
+.SH DESCRIPTION
+upload is an sos subcommand to upload sos reports, logs, vmcores, or other files to a policy defined remote location, or an user defined one.
```

Comment:
Nitpick: `s/an user/a user/`.

Reply:
I think is 'an' because 'user' starts with a vowel, no?

Reply:
Nice, TIL. Fixed in the next push.